corehq/tabs/config.py

```diff
@@ -19,6 +34,7 @@
     MySettingsTab,
     TranslationsTab,
     HostedCCZTab,
+    LocationRationalizationTab,
```

Comment:
Why isn't this under the Organizations section in the existing Users tab?

Reply:
To add a bit to this: we should avoid introducing new tabs. There are a bunch of them in app manager because app manager's left-hand menu is unique. Outside of app manager, adding new pages to the left-hand menu is almost always the right decision.

Reply:
Is it under the Organizations section in the existing Users tab and is visible only if feature flag is enabled.

Reply:
Sorry, first comment wasn't clear. This feature shouldn't show up in the top navigation, which is only for the most frequently-used pages. It should be part of the existing `ProjectUsersTab`. Each item in the top nav should only have one "tab" in this file, to make it easier for users to understand how to navigate the site. There are exceptions to this, but I don't see why this feature should be an exception.

Reply:
Resolved by https://github.com/dimagi/commcare-hq/pull/26892